src/myapps/organising-download-dir.py

```diff
@@ -0,0 +1,119 @@
+# automation.py
+import os
+from os.path import isfile, join
+from pathlib import Path
+from shutil import move
+from typing import List
+import logging
+
+# directory paths
+
+BASE_PATH = Path(os.path.abspath(__file__)).parent.parent.parent.parent.parent.parent
+TARGET_PARENT = "Downloads"  # change target location here; e.g. /home/username/{foobar}
+TARGET_PARENT_PATH = f"{BASE_PATH}/{TARGET_PARENT}"
+
+parent_images_path = f"{TARGET_PARENT_PATH}/images"
+parent_documents_path = f"{TARGET_PARENT_PATH}/documents"
+parent_others_path = f"{TARGET_PARENT_PATH}/others"
+parent_softwares_path = f"{TARGET_PARENT_PATH}/softwares"
+
+# category wise file types
+
+doc_types = (".doc", ".docx", ".txt", ".pdf", ".xls", ".ppt", ".xlsx", ".pptx")
+img_types = (".jpg", ".jpeg", ".png", ".svg", ".gif", ".tif", ".tiff")
+software_types = (".exe", ".pkg", ".dmg", "deb")
+
+
+def get_files(parent_dir) -> List:
+    """Returns a list containing every filenames in the
+    `parent_dir`. Hidden or dot files are ignored
+
+    Arguments:
+        parent_dir {string} -- base path
+
+    Returns:
+        List -- Every filenames found in `parent_dir` (e.g. 'foobar.txt')
+    """
+    return [
+        f
+        for f in os.listdir(parent_dir)
+        if isfile(join(parent_dir, f))
+        and not f.startswith(".")
+        and not f.__eq__(__file__)
+    ]
+
+
+def create_directories(image_path, documents_path, software_path, others_path):
```

Comment:
Do you expect these parameters to ever change? Even if you do, why do we need to pass them as parameters? Instead of having global variables on lines 15-18, could we not just define the paths in this function?